### ISSUE TITLE
WASM : Cannot find name 'EmscriptenModule'

### ISSUE THREAD

**Issue by sbstcano** [tensorflow/tfjs#3866]
Hi,
I can't compile my Angular 9 project because of this error.
I'm using version 2.3.0 of wasm.

```
ERROR in node_modules/@tensorflow/tfjs-backend-wasm/wasm-out/tfjs-backend-wasm.d.ts:18:44 - error TS2304: Cannot find name 'EmscriptenModule'.

18 export interface BackendWasmModule extends EmscriptenModule {
```

**Comment by rthadur** [comment]
@sbstcano did you install  Emscripten from here https://github.com/tensorflow/tfjs/tree/master/tfjs-backend-wasm#emscripten-installation ?

**Comment by sbstcano** [comment]
> @sbstcano did you install Emscripten from here https://github.com/tensorflow/tfjs/tree/master/tfjs-backend-wasm#emscripten-installation ?

Yes, I tried and it doesn't help

**Comment by rthadur** [comment]
@sbstcano did you try with previous version ?

**Comment by sbstcano** [comment]
> @sbstcano did you try with previous version ?

Yes and sadly it doesn't help. I tried with 2.1.0, 2.0.1, 1.7.4. I tried to downgrade my angular version to ^8. I tried ^10. Angular uses es5 and webpack. Can it be related?

**Comment by sivarajakani** [comment]
I also have this issue, you found a way to handle this issue?

**Comment by rthadur** [comment]
@jinjingforever gentle ping if you got chance to look in to this issue, thank you
